examples/avm/addressFromBuffer.ts

```diff
@@ -30,10 +30,13 @@ const main = async (): Promise<any> => {
   const utxos: UTXO[] = utxoset.getAllUTXOs()
 
   utxos.map((utxo: UTXO): void => {
-    const output: BaseOutput = utxo.getOutput()
```

Comment:
Why?

Reply:
probably an accidental change from an avax commit which has been reverted later on. I will revert it.